### ISSUE TITLE
Integrate with core's "pageless node" concept?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/rabbit_hole#11]
There's now a bit of duplication as core provides "pageless node" capability: https://github.com/backdrop/backdrop-issues/issues/100

Maybe there's a way to integrate that bit of core functionality with Rabbit Hole so we don't duplicate it. I haven't investigated too closely yet.

**Comment by laryn** [comment]
Here's the core functionality as described:

> If enabled, this content can be placed as a block or within a listing but will not be accessible at its URL without proper permissions. This facilitates "hidden path" content that is meant to be embedded with other content and never on its own. Users who have the "View hidden paths" permission will still be able to visit the full page URL, while those without this permission will receive a "Page not found" error.

We'd need to figure out for Rabbit Hole nodes whether or how to check (and set?) this core value, and then recognize that by default the core setting only allows the 404 response, not the other RH options that are typically allowed. If the core setting is checked, we could configure Rabbit Hole nodes to the corresponding "Page not found" behaviour and then lock it from direct editing, perhaps?

So to summarize one possibility:

 - If the core "Hide path display" is set under "Display settings" on a node form:
 - Disable the settings under "Rabbit Hole" for that node type, forcing selection of the "Page not found" behaviour.
 - Add a note if possible indicating why things are disabled.
